lib/Controller/CallController.php

```diff
@@ -63,7 +62,6 @@ public function __construct($appName,
 								IRequest $request,
 								IUserManager $userManager,
 								ISession $session,
-								ILogger $logger,
 								Manager $manager,
 								Messages $messages) {
```

Comment:
messages is also only used in the constructor, dare to remove it?

Reply:
Done